rotkehlchen/db/upgrades/v46_v47.py

```diff
@@ -110,53 +110,56 @@ def _remove_nft_collection_assets(write_cursor: 'DBCursor') -> None:
         with the same address that were incorrectly added.
         """
         with (globaldb_conn := GlobalDBHandler().conn).read_ctx() as global_db_cursor:
-            identifiers_to_remove = tuple(entry[0] for entry in global_db_cursor.execute(
+            globaldb_identifiers_to_remove = tuple(entry[0] for entry in global_db_cursor.execute(
                 'SELECT identifier FROM assets WHERE identifier IN ('
                 "SELECT identifier FROM evm_tokens WHERE token_kind = 'B' "
                 "UNION SELECT REPLACE(identifier, 'erc721', 'erc20') "
                 "FROM evm_tokens WHERE token_kind = 'B') AND "
                 "LENGTH(identifier) <= 62;",  # Skip identifiers with collectible ids (will be longer than 62)  # noqa: E501
```

Comment:
```suggestion
                "identifier NOT LIKE 'eip155:%/erc721:0x%/%'",  # Skip identifiers with collectible ids  # noqa: E501
```
Can't check for length or it will skip tokens on other chains even if they don't have a token id as we discussed in discord.